src/ut_control_plane.c

```diff
@@ -262,7 +273,81 @@ static int callback_echo(struct lws *wsi, enum lws_callback_reasons reason, void
 
     return 0;
 }
+#else
+static int callback_http(struct lws *wsi, enum lws_callback_reasons reason, void *user, void *in, size_t len)
+{
+    cp_message_t msg;
+    ut_cp_instance_internal_t *pInternal = (ut_cp_instance_internal_t* )lws_context_user(lws_get_context(wsi));
+    struct per_session_data__http *pss = (struct per_session_data__http *)user;
+
+    switch (reason)
+    {
+        case LWS_CALLBACK_HTTP: {
+            UT_CONTROL_PLANE_DEBUG("LWS_CALLBACK_HTTP\n");
+            char *requested_uri = (char *)in;
+
+            if (strcmp(requested_uri, "/api/postKVP") == 0) {
+                lws_callback_on_writable(wsi);
+                return 0;
+            } else {
+                lws_serve_http_file(wsi, "index.html", "text/html", NULL, 0);
```

Comment:
where does this file come from?

Reply:
was not required, so removed that as well.